docs/notebooks/ppg/ppg_analysis.ipynb

```diff
@@ -65,30 +86,15 @@
     "metadatas_ppg, metadatas_imu = scan_and_sync_segments(os.path.join(path_to_sensor_data, 'ppg'),\n",
     "                                                       os.path.join(path_to_sensor_data, 'imu'))\n",
     "preprocess_ppg_data(metadatas_ppg[0], metadatas_imu[0],\n",
```

Comment:
Waarom gebruik je hier alleen de 1e index? Als je dit vaker wilt gebruiken zou je de functie ook wellicht kunnen draaien met alleen de 1e index als uitkomst, in plaats van voor alle data te draaien en dan alleen de 1e index te pakken?